config/mappings/test_mapping_config.py

```diff
@@ -1,7 +1,11 @@
-import json
 from jsonschema import validate
-from assessment_mapping_schema import top_level_assessment_mapping_schema as schema
-from assessment_mapping import cof_r2w2_assessment as mapping
 
-validate(instance=mapping, schema=schema)
-print("Mapping conforms with schema.")
+from .assessment_mapping import cof_r2w2_assessment as mapping
+from .assessment_mapping_schema import (
+    top_level_assessment_mapping_schema as schema,
+)
+
+
+def test_assessment_mapping_schema():
+    validate(instance=mapping, schema=schema)
+    print("Mapping conforms with schema.")
```

Comment:
⛏️ Use a assert here. Validate raises an error if the the schema doesn't validate. Also pytest doesn't print strings by default.

Reply:
Yeah good shout, have also moved this to the `tests` dir